tools/data_exploration/stat_presence_abs.xml

```diff
@@ -17,6 +17,7 @@
             '$species'
             '$location'
             '$time'
+            '$__tool_directory__/functions.r'

```

Comment:
what does this script do? Its just a function and is not triggered isn't it?

Reply:
It's a script of a function that I use several times. I call it in graph_lcbd.r and in graph_stat_presence_abs.r.